packages/CodeGenLib/src/Database/manage-metadata.ts

```diff
@@ -1633,9 +1633,10 @@ export class ManageMetadataBase {
     * @param ds - The DataSource object to use to execute the query.
     * @param query - The SQL query to execute.
     * @param description - A description of the query to append to the log file.
+    * @param isRecurringScript - if set to true tells the logger that the provided SQL represents a recurring script meaning it is something that is executed, generally, for all CodeGen runs. In these cases, the Config settings can result in omitting these recurring scripts from being logged because the configuration environment may have those recurring scripts already set to run after all run-specific migrations get run.
```

Comment:
Doesn't really matter, but just FYI the flyway `R` stands for Repeatable– meaning it's fine to re-run the script